contracts/lupi.sol

```diff
@@ -3,9 +3,14 @@ pragma solidity ^0.4.11;
 import "./Owned.sol";
```

Comment:
This version of Owned.sol is used in a lot of unofficial examples/tutorials.
A slightly different one is used as an example in official solidity docs:
http://solidity.readthedocs.io/en/develop/contracts.html#function-modifiers